### PR TITLE
pass in check keys option to allow "." in keys

### PR DESCRIPTION
**Pull request by andyh565**
…_Model.create

### CODE REVIEW

**Comment by andyh565** [comment]
documentation: https://mongoosejs.com/docs/api.html#model_Model.create

File: packages/hull-vm/server/save-recent.js [hunk 1]
```diff
@@ -26,5 +26,7 @@ export default function saveRecent(
     payload,
     date: new Date().toString()
   };
-  return EntryModel.create(entry);
+  return EntryModel.create([entry], {
```
Comment (andyh565) [inline packages/hull-vm/server/save-recent.js:29]: entry must be an array according to the documentation:
https://mongoosejs.com/docs/api.html#model_Model.create

**Comment by unity** [comment]
Seems ok to me. Haven’t tested the behavior or side effects doing this. But it seems like a good straightforward fix

File: packages/hull-vm/server/save-recent.js [hunk 2]
```diff
@@ -26,5 +26,7 @@ export default function saveRecent(
...
+    checkKeys: false
+  });
```
Comment (andyh565) [inline packages/hull-vm/server/save-recent.js:31]: entry must be an array according to the documentation:
https://mongoosejs.com/docs/api.html#model_Model.create

Comment (unity) [reply packages/hull-vm/server/save-recent.js:31]: It seemed to work before - do we know why ?

**Comment by andyh565** [comment]
> Seems ok to me. Haven’t tested the behavior or side effects doing this. But it seems like a good straightforward fix

okay great. The fix may be a little more complicated for the legacy connector since I'll need to update the mongoose version as well

File: packages/hull-vm/server/save-recent.js (hunk 2 above)
Comment (andyh565) [reply packages/hull-vm/server/save-recent.js:31]: I'm not sure. I was able to reproduce the error the first time by adding a "." in one of the keys

Comment (andyh565) [reply packages/hull-vm/server/save-recent.js:31]: > It seemed to work before - do we know why ?

yes, it only needs to be an array if you pass in the options

**Pull request merged**